src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmethod/InputJavadocMethodExtraThrows.java

```diff
@@ -12,7 +16,7 @@
      * @throws IllegalArgumentException when argument is wrong // ok
      * @throws NullPointerException indicates null was passed // ok
      */
-    public InputJavadocMethodExtraThrows(String properties) {
+    public InputJavadocMethodExtraThrows(String properties) {                                 // ok
```

Comment:
I am not sure such alignment at the end is good.
especially when all others are not following this.

please remove extra spaces between code and this traililng comment

Reply:
Such alignment looks a bit disciplined, also moving onto to BDD, visual inspection would be important while reviewing, and in our alignment we might miss some. I have removed right alignment in this PR, but we can follow this approach in next updates.